README.md

```diff
@@ -317,10 +333,18 @@ You can access a vector of pointers to the parsed options in the original order
 If `--` is present in the command line that does not end an unlimited option, then
 everything after that is positional only.
 
+#### Getting results
+In most cases the easiest way is to return the results through a callback or variable specified in one of the `add_*` functions.  But there are situations where this is not possible or desired.  For these cases the results may be obtained through one of the following functions
+
+- `results()`: retrieves a vector of strings with all the results in the order they were given.
+- `results(variable_to_bind_to)`: gets the results according to the MultiOptionPolicy and converts them just like the `add_option_function` with a variable.
```

Comment:
This and the ones below it will "reconvert" the variable from a string; this should not be done in performance critical code. A note of some sort might be useful here or nearby.